core/src/main/java/de/valtech/aecu/core/groovy/console/bindings/actions/resource/MoveResourceToRelativePath.java

```diff
@@ -33,10 +38,18 @@
 
     private String relativePath;
     private ResourceResolver resourceResolver;
+    private boolean dryRun;
 
-    public MoveResourceToRelativePath(@Nonnull String relativePath, @Nonnull ResourceResolver resourceResolver) {
+    /**
+     * Constructor
+     * 
+     * @param relativePath relative path
+     * @param context      binding context
+     */
+    public MoveResourceToRelativePath(@Nonnull String relativePath, @Nonnull BindingContext context) {
         this.relativePath = relativePath;
-        this.resourceResolver = resourceResolver;
+        this.resourceResolver = context.getResolver();
+        dryRun = context.isDryRun();
```

Comment:
Same here: Keep the context reference as class attribute because dryRun is set later in the call stack, else dryRun will be always true.

Reply:
fixed